src/DatePickerRange/DatePickerRange.component.tsx

```diff
@@ -0,0 +1,167 @@
+import React from 'react';
+import styled, { ThemeProvider } from 'styled-components';
+import { DateRangePicker, DateRangePickerShape } from 'react-dates';
+import uuid from 'uuid';
+import { Moment } from 'moment';
+
+type momentDate = Moment | null;
+type pickerSize = 'sm' | 'md' | 'lg';
+type pickerStyle =
+  | 'default'
+  | 'primary'
+  | 'secondary'
+  | 'success'
+  | 'warning'
+  | 'danger';
+type focusInputs = FocusInputs.START_DATE | FocusInputs.END_DATE | null;
+enum FocusInputs {
+  START_DATE = 'startDate',
+  END_DATE = 'endDate',
+}
+type dateChangeEvent = { startDate: momentDate; endDate: momentDate };
+
+export interface Props extends DateRangePickerShape {
+  /**
+   * Set className
+   *
+   * @default ''
+   **/
+  className?: string;
+  /**
+   * The ID of the control
+   *
+   * @default null
+   **/
+  id?: string;
+  /**
+   * Select DatePicker Size
+   *
+   * @default 'md'
+   **/
+  datePickerSize?: pickerSize;
+  /**
+   * Select DatePicker style.
+   *
+   * @default 'primary'
+   **/
+  datePickerStyle?: pickerStyle;
+  /**
+   * From theme provider
+   *
+   * @default defaultTheme
+   **/
+  theme?: any;
+}
+
+type State = {
+  focusedInput: focusInputs;
+  range: dateChangeEvent;
+};
+
+const SWrapperComponent = styled.div<any>`
+  margin-top: ${(props: any) =>
+    props.theme.common[props.datePickerSize!].padding.toString().split(' ')[1]};
+  font-family: ${(props: any) => props.theme.typography.fontFamily};
+  font-size: ${(props: any) =>
+    props.theme.common[props.datePickerSize!].fontSize}
+  color: ${(props: any) => props.theme.colors.primary};
+  box-shadow: #0000001a 0px 3px 25px;
```

Comment:
These styles need to be adjusted. No `box-shadow` or `position` needed on the input. Are these meant to be applied to the popover box?

Reply:
Removed and few other lines.